core/server/services/xmlrpc.js

```diff
@@ -67,7 +67,7 @@ function ping(post) {
             timeout: 2 * 1000
         };
 
-        const goodResponse = /<member><name>flerror<\/name><value><boolean>0<\/boolean><\/value><\/member>/;
+        const goodResponse = /<member>[\s]*<name>flerror<\/name>[\s]*<value>[\s]*<boolean>0<\/boolean><\/value><\/member>/;
```

Comment:
Was toying with `+` and `*` here, pushed the `+` one before. We _might_ wanna use `*` instead of `+` in the below too, but the message does always include whitespace so IDK

Reply:
+ is one or more
* is zero or more

So * makes sense to me here 👍